src/compaction.c

```diff
@@ -30,6 +31,7 @@ typedef struct AvgContext
 {
     double val;
     double cnt;
+    unsigned char isOverflow;
```

Comment:
```suggestion
    bool isOverflow;
```

Reply:
@danni-m So why here char is ok?
typedef struct MaxMinContext
{
    double minValue;
    double maxValue;
    char isResetted;
} MaxMinContext;